nototools/family_name_info_p3.xml

```diff
@@ -1,123 +1,120 @@
-  <info family="Noto Serif Georgian" limit_original="t" />
-  <info family="Noto Serif Gujarati" />
-  <info family="Noto Serif Kannada" />
-  <info family="Noto Serif Khmer" />
-  <info family="Noto Serif Lao" />
-  <info family="Noto Serif Malayalam" />
-  <info family="Noto Serif Tamil" />
-  <info family="Noto Serif Telugu" />
-  <info family="Noto Serif Thai" />
-  <info family="Tinos" />
+<family_name_data date="2016-03-11">
+  <info family="arimo-lgc" />
+  <info family="cousine-lgc" />
+  <info family="emoji-zsye" />
+  <info family="emoji-zsye-color" />
+  <info family="kufi-arab" />
+  <info family="mono-mono" use_preferred="t" use_wws="t" />
+  <info family="naskh-arab" use_preferred="t" use_wws="t" />
+  <info family="nastaliq-aran" />
+  <info family="sans-armi" />
+  <info family="sans-armn" limit_original="t" />
+  <info family="sans-avst" />
+  <info family="sans-bali" />
+  <info family="sans-bamu" />
+  <info family="sans-batk" />
+  <info family="sans-beng" use_preferred="t" use_wws="t" />
+  <info family="sans-brah" />
+  <info family="sans-bugi" />
+  <info family="sans-buhd" />
+  <info family="sans-cans" />
+  <info family="sans-cari" />
+  <info family="sans-cham" />
+  <info family="sans-cher" />
+  <info family="sans-cjk-jp" limit_original="t" />
+  <info family="sans-cjk-kr" limit_original="t" />
+  <info family="sans-cjk-sc" limit_original="t" />
+  <info family="sans-cjk-tc" limit_original="t" />
+  <info family="sans-copt" />
+  <info family="sans-cprt" />
+  <info family="sans-deva" limit_original="t" use_wws="t" />
+  <info family="sans-dsrt" />
+  <info family="sans-egyp" />
+  <info family="sans-ethi" />
+  <info family="sans-geor" limit_original="t" />
+  <info family="sans-glag" />
+  <info family="sans-goth" />
+  <info family="sans-gujr" use_preferred="t" use_wws="t" />
+  <info family="sans-guru" use_preferred="t" use_wws="t" />
+  <info family="sans-hano" />
+  <info family="sans-hans" limit_original="t" />
+  <info family="sans-hant" limit_original="t" />
+  <info family="sans-hebr" />
+  <info family="sans-ital" />
+  <info family="sans-java" />
+  <info family="sans-jpan" limit_original="t" />
+  <info family="sans-kali" />
+  <info family="sans-khar" />
+  <info family="sans-khmr" use_preferred="t" use_wws="t" />
+  <info family="sans-knda" use_preferred="t" use_wws="t" />
+  <info family="sans-kore" limit_original="t" />
+  <info family="sans-kthi" />
+  <info family="sans-lana" />
+  <info family="sans-laoo" use_preferred="t" use_wws="t" />
+  <info family="sans-lepc" />
+  <info family="sans-lgc" limit_original="t" use_wws="t" />
+  <info family="sans-limb" />
+  <info family="sans-linb" />
+  <info family="sans-lisu" />
+  <info family="sans-lyci" />
+  <info family="sans-lydi" />
+  <info family="sans-mand" />
+  <info family="sans-mlym" use_preferred="t" use_wws="t" />
+  <info family="sans-mong" />
+  <info family="sans-mono-mono" limit_original="t" use_wws="t" />
```

Comment:
I noticed that "sans-mono-mono" is only in this file and not the other, is that intentional?